web/modules/custom/dpl_paragraphs/dpl_paragraphs.module

```diff
@@ -223,6 +223,7 @@ function dpl_paragraphs_preprocess_paragraph__material_grid_automatic(array &$va
 
   // Extract field values.
   $material_grid_title = $paragraph->get('field_material_grid_title')->getString();
+  $material_grid_description = $paragraph->get('field_material_grid_description')->getString();
```

Comment:
Please add a check to see if `field_material_grid_description` exists (`hasField`)

Otherwise, this will fail hard on existing paragraphs

Reply:
Yeah good catch 👍